src/OrchardCore/Orchard.Mvc.Core/Extensions/ModularServiceCollectionExtensions.cs

```diff
@@ -62,6 +61,13 @@ public static class ModularServiceCollectionExtensions
             return services;
         }
 
+        public static void AddTagHelpers(this IServiceProvider serviceProvider, string assemblyName)
```

Comment:
We should have an overload with an Assembly as parameter. This way we can just use `typeof(MyTagHelper).Assembly`